### PR TITLE
Extract language_dump.py logic into functions

### PR DESCRIPTION
**Pull request by tupaschoal**
~*This is in draft while #76 is not merged, as it builds on top of it*~

This extracts `language_dump.py` logic into functions and also conforms to `pylint`

### CODE REVIEW

File: language_dump.py [hunk 1]
```diff
@@ -1,37 +1,41 @@
 #!/usr/bin/env python3
+"""
+Dump translations for OpenRCT2\'s JSON objects given a language and a file
```
Comment (AaronVanGeffen) [inline language_dump.py:3]: The apostrophe doesn't need to be escaped here, right?
```suggestion
Dump translations for OpenRCT2's JSON objects given a language and a file
```

File: language_dump.py [hunk 2]
```diff
@@ -40,16 +44,36 @@
...
                 if verbose:
                     print("Found existing translation for " + data['id'])
-                strings_by_object[data['id']][string_key] = data['strings'][string_key][language_to_extract]
+                strings_by_object[data['id']][string_key] = \
+                    data['strings'][string_key][language_to_extract]
             elif fallback_language in data['strings'][string_key]:
-                print("No existing translation for " + data['id'] + " yet, using English")
-                strings_by_object[data['id']][string_key] = data['strings'][string_key][fallback_language]
+                print("No existing translation for " + data['id'] + \
+                      " yet, using English")
```
Comment (AaronVanGeffen) [inline language_dump.py:51]: This might be a good opportunity to use f-strings.

File: language_dump.py [hunk 3]
```diff
@@ -40,16 +44,36 @@
             if language_to_extract in data['strings'][string_key]:
                 if verbose:
                     print("Found existing translation for " + data['id'])
-                strings_by_object[data['id']][string_key] = data['strings'][string_key][language_to_extract]
+                strings_by_object[data['id']][string_key] = \
+                    data['strings'][string_key][language_to_extract]
```
Comment (AaronVanGeffen) [inline language_dump.py:48]: This feels a little awkward due to the wrapping. Perhaps save the rvalue first with a local (alias) assignment.

File: language_dump.py [hunk 4]
```diff
@@ -40,16 +44,36 @@
...
+                    data['strings'][string_key][fallback_language]
             else:
                 if verbose:
-                    print("No existing translation for " + data['id'] + " yet, but no English string either -- skipping")
-                # pprint(data)
+                    print("No existing translation for " + data['id'] + \
+                          " yet, but no English string either -- skipping")
+
+def dump_translations():
+    """ Dump translations for OpenRCT2\'s JSON objects """
```
Comment (AaronVanGeffen) [inline language_dump.py:60]:
```suggestion
    """ Dump translations for OpenRCT2's JSON objects """
```

File: language_dump.py (hunk 2 above)
Comment (tupaschoal) [reply language_dump.py:51]: I didn't know about that, will do!

**Comment by tupaschoal** [comment]
I solved the issues and also removed some hardcoded `"English"` where it should actually be the fallback language

**Comment by tupaschoal** [comment]
I did!

**Pull request merged**